plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3select/S3SelectPushdown.java

```diff
@@ -106,12 +106,10 @@ private static boolean isInputFormatSupported(Properties schema)
     public static boolean isCompressionCodecSupported(InputFormat<?, ?> inputFormat, Path path)
     {
         if (inputFormat instanceof TextInputFormat) {
-            var compressionCodec = getCompressionCodec((TextInputFormat) inputFormat, path);
-
             // S3 Select supports the following formats: uncompressed, GZIP and BZIP2.
-            return compressionCodec.isEmpty() || compressionCodec
```

Comment:
This should be done right already in the `Enable Select pushdown on uncompressed files` commit